hacking/hacking.md

```diff
@@ -50,26 +50,19 @@ This will result in the following binary files:
 ## Alternative Hardware
 
 A number of users have contributed alternative hardware designs that are compatible with the native firmware.
-Some of these designs are in the [hardware repo](https://github.com/blackmagic-debug/blackmagic-hardware/tree/master/contrib). Check the `README.md` files for details. For instance, to compile a BMP for an ST-Link v2 to run as alternative to the ST firmware, compile:
+Some of these designs are in the [hardware repo](https://github.com/blackmagic-debug/blackmagic-hardware/tree/master/contrib). Check the `README.md` files for details. For instance, to compile a BMP for an ST-Link V2 to run as alternative to the ST firmware, compile:
 
 ```bash
 make PROBE_HOST=stlink ST_BOOTLOADER=1
 ```
 
-then replug the STLink to get into the bootloader and load with [stlink-tools] (<https://github.com/jeanthom/stlink-tool>)
-
-```bash
-stlink-tools blackmagic.bin
-```
-
 ## Building on Windows
 
 Sid Price wrote a detailed step by step guide describing [how to set up CygWin and compile the Black Magic Probe firmware](http://www.sidprice.com/2018/05/23/cortex-m-debugging-probe/).
 
 ### Compiling as a PC application
 
-The Black Magic application can also be compiled as a native PC application.
-Supported probes are BMP (with recent firmware), ST-Link v2/v3, FTDI-MPSSE probes, JLink and CMSIS-DAP.
+The Black Magic Debug Application can also be compiled as a native PC application.
```

Comment:
```suggestion
The Black Magic Debug project can also be compiled as a desktop program named Black Magic Debug App.
When compiled in this way it can support using the following adaptor types to talk with your hardware:

* BMP with a firmware version of at least v1.6.2
* ST-Link v2 and v3
* FTDI-MPSSE devices
* J-Link probes
* CMSIS-DAP complaint probes
```

The second part of this might do better moved, as you've done, to hardware.md - we would still suggest the rewording about what BMDA is as we want to move away from Uwe's description of it as "a native PC application" - it's a desktop computer program.

Reply:
Implemented line 65 as you suggested.